Adding FCA UK Chapter 15 Operational resilience framework

jledoze opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Introduced a comprehensive operational resilience framework for UK regulated firms featuring a detailed scoring system that rates resilience from 1 (Initial) to 5 (Optimizing).
	- Provided clear guidelines on critical business services identification, impact tolerances, scenario testing, and structured compliance reviews to support robust risk management.
	- Added essential metadata for improved clarity and reference regarding operational resilience regulations.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

jledoze commented:
I have read the CLA Document and I hereby sign the CLA

eric-intuitem commented:
1/ A license agreement must be obtained: https://www.fca.org.uk/handbook-licence-agreement.
2/ The IG tab is not used. Either it should be removed, or preferably it should be used to distinguish "Rules" from "Guidance". R and G should be visible also in the title.
3/ Is the score system part of the standard and/or relevant?

jledoze commented:
Hello, 
1/ About the licence agreement, could you ask for it? If it is added, it would be in your product. So, it makes more sens for you to ask for it no? 
2/ Thank you for the recommandation. I added it. About the R and G in the title, do you mean it should appear in the "ref_id"? 
3/ The score is not a part of the standard. I added it because CISO-Assistant allows to add it. I believe this is a nice to have. 
4/ Done

eric-intuitem commented:
License request sent.
